lib/srv/reexec.go

```diff
@@ -78,8 +78,6 @@ const (
 	// to pid 1 and "live forever". Killing the shell should not prevent processes
 	// preventing SIGHUP to be reassigned (ex. processes running with nohup).
 	TerminateFile
-	// PTYFile is a PTY the parent process passes to the child process.
-	PTYFile
```

Comment:
Not that we care that much about it, but changing the "interface" of Teleport reexec like this has the potential to cause really weird issues during Teleport upgrades on darwin, because of the confusion between `PTYFile` and `TTYFile`.

Can we leave PTYFile there and just not populate it? A `nil` `*os.File` in `ExtraFiles` will be left closed in the child.